src/components/DayOverview/DayOverviewPdf.tsx

```diff
@@ -0,0 +1,205 @@
+"use client";
+
+import React from "react";
+import { Svg, Document, Page, Text, View, StyleSheet, Path } from "@react-pdf/renderer";
+import { faFlag, faSquare, IconDefinition } from "@fortawesome/free-solid-svg-icons";
+
+import { ParcelOfSpecificDateAndLocation } from "@/components/DayOverview/DayOverview";
+
+interface DayOverviewDateProps {
+    date: Date;
+}
+
+interface DayOverviewLocationProps {
+    location: string;
+}
+
+interface DayOverviewRowProps {
+    row: ParcelOfSpecificDateAndLocation;
+}
+
+interface DayOverviewPdfProps {
+    date: Date;
+    location: string;
+    data: ParcelOfSpecificDateAndLocation[];
+}
+
+interface DayOverviewCardProps extends DayOverviewPdfProps {}
+
+interface CustomSVGProps {
+    icon: IconDefinition;
+    color: string;
+    fill: boolean;
+}
+
+const styles = StyleSheet.create({
+    page: {
+        display: "flex",
+        flexDirection: "column",
+        fontSize: "11pt",
+    },
+    card: {
+        margin: "0 30pt 15pt",
+    },
+    table: {
+        borderStyle: "solid",
```

Comment:
Is there borderLeftStyle? Not sure if there is tho so please ignore if not! (In which case could be added inline)

Reply:
There is borderLeft. Refactored as one-line (also the same for `styles.table`!)